### DISCUSSION TITLE
Unified colors in the list

### DISCUSSION THREAD

**Discussion by cywil5115** [linkarzu/dotfiles-latest#3] [General]
Hi,

I really like your colorscheme script. I've been trying to add extra programs to use selected colors but it's really hard to select right hex for many different configs and themes format. I am looking for a discussion, if it's possible to unify theme creation and setting it up. Just the framework of these colors and in the script itself. I've managed to change needed paths.

**Comment by linkarzu** [comment]
Hey there, sorry, not following, can you explain again, you want to change something or you would like for me to change something?
